tools/build/libwebrtc/lib.sh

```diff
@@ -0,0 +1,275 @@
+function create-WORK_DIR-and-cd() {
+    # Ensure directory exists
+    mkdir -p $WORK_DIR
+    WORK_DIR=$(cd $WORK_DIR && pwd -P)
+    # cd to directory
+    cd $WORK_DIR
+    # mkdir webrtc subdir
+    mkdir -p $SRC_DIR
+}
+
+#-----------------------------------------------------------------------------
+function verify-host-platform-deps() {
+    # TODO check for required software packages, environment variables, etc.
+    :
+}
+
+#-----------------------------------------------------------------------------
+function checkout-depot-tools() {
+    echo -e "\e[39mCloning depot tools -- this may take some time\e[39m"
+    if [ ! -d $DEPOT_TOOLS_DIR ]; then
+        git clone -q $DEPOT_TOOLS_URL $DEPOT_TOOLS_DIR
+        if [ $HOST_OS = 'win' ]; then
+            # run gclient.bat to get python
+            pushd $DEPOT_TOOLS_DIR >/dev/null
+            ./gclient.bat
+            popd >/dev/null
+        fi
+    else
+        pushd $DEPOT_TOOLS_DIR >/dev/null
+        git reset --hard -q
+        popd >/dev/null
+    fi
+}
+
+#-----------------------------------------------------------------------------
+function checkout-webrtc() {
+    echo -e "\e[39mCloning WebRTC source -- this may take a long time\e[39m"
+    pushd $SRC_DIR >/dev/null
+
+    # Fetch only the first-time, otherwise sync.
+    if [ ! -d src ]; then
+        case $TARGET_OS in
+        android)
+            yes | fetch --nohooks webrtc_android
+            ;;
+        ios)
+            fetch --nohooks webrtc_ios
+            ;;
+        *)
+            fetch --nohooks webrtc
+            ;;
+        esac
+    fi
+    # Checkout the specific revision after fetch.
+    gclient sync --force --revision $REVISION
+
+    popd >/dev/null
+}
+
+#-----------------------------------------------------------------------------
+function latest-rev() {
+    git ls-remote $REPO_URL HEAD | cut -f1
+}
+
+#-----------------------------------------------------------------------------
+function calc-git-revision() {
+    echo -e "\e[39mFetching Git revision\e[39m"
+    if [ ! -z $BRANCH ]; then
+        REVISION=$(git ls-remote $REPO_URL --heads $BRANCH | head -n1 | cut -f1) ||
+            { echo -e "\e[91mCound not get branch revision for $BRANCH\e[39m" && exit 1; }
+        [ -z $REVISION ] && echo "Cound not get branch revision for $BRANCH\e[39m" && exit 1
+    else
+        REVISION=${REVISION:-$(latest-rev $REPO_URL)} ||
+            { echo -e "\e[91mCould not get latest revision\e[39m" && exit 1; }
+    fi
+    echo -e "\e[39mRevision: \e[96m$REVISION\e[39m"
+}
+
+#-----------------------------------------------------------------------------
+function verify-webrtc-deps() {
+    echo -e "\e[39mVerifying WebRTC dependencies. You may be prompted to accept software licenses.\e[39m"
+    case $HOST_OS in
+    "linux")
+        echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | sudo debconf-set-selections
```

Comment:
This needs to be changed, we can't accept an EULA for the user.

Reply:
Actually this was supposed to be fixed, looking through this PR comments above. Did that cropped up again later?

Reply:
Oh I thought I got rid of that already.

Reply:
Deleted.